ethereum/src/logs_handler.rs

```diff
@@ -0,0 +1,21 @@
+use crate::{
+	Result,
+	EthereumLikeChainClient
+};
+use async_trait::async_trait;
+use web3::types::{Log, H160, H256};
+
+#[async_trait]
+pub trait LogsHandler {
```

Comment:
Rename `LogsHandler` to `ChainLogHandler`. Cause `LogsHandler`  not telling what kind of log we are dealing with.